app/models/concerns/taggable.rb

```diff
@@ -61,7 +61,7 @@ def current_tags_for_language(language_id)
 
     language = Language.find(language_id)
 
-    tags_on(language_code)
+    tags_on(language.code.to_sym)
```

Comment:
@Oli0li maybe you know
Did we want here to get tags from this language passed by `language_id`?

Reply:
I think we'll remove this method later when we make sure everything else you've mentioned is deleted as well